install.yml

```diff
@@ -62,4 +64,4 @@
   - { role: dns, tags: [ 'dns' ] }
   - { role: ansible-role-collectd, tags: [ 'collectd', 'monitoring' ] }
   - { role: ansible-role-flowdock, tags: [ 'flowdock', 'always' ] }
-
+  - { role: ansible-role-sshd-host-keys-client, tags: [ 'ssh', 'ssh-hostca' ] }
```

Comment:
No, the second time is when running against admin after running against install. Can't put the host-keys-client role directly into admin.yml because then it would fail before the server part is setup on install.